staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -175,6 +176,9 @@ type Config struct {
 	// if the client requests it via Accept-Encoding
 	EnableAPIResponseCompression bool
 
+	// RuntimeConfig contains the options for which resources to turn on and off.
+	RuntimeConfig utilflag.ConfigurationMap
```

Comment:
This should move into `APIEnablementOptions` (or the corresponding `ApplyTo` func) too. Here we should have a complete, merged `*ResourceConfig`. Flag processing belongs into the options.